packages/widgets/src/HeaderBar/TextIcon.js

```diff
@@ -1,19 +1,20 @@
 import propTypes from '@dhis2/prop-types'
 import React from 'react'
 
-export const TextIcon = ({ name, onClick, dataTestId }) => {
-    let title = name[0]
-
-    if (name.indexOf(' ') > 0) {
-        title += name.split(' ')[1][0]
-    }
+export const getInitials = name =>
+    name
+        .split(' ')
+        .slice(0, 2)
+        .map(n => n[0])
+        .join('')
 
+export const TextIcon = ({ name, onClick, dataTestId }) => {
     return (
-        <div onClick={onClick} data-test={dataTestId}>
-            <p>{title}</p>
+        <div className={'text-icon'} onClick={onClick} data-test={dataTestId}>
```

Comment:
Is there a reason to use `{'text-icon'}` instead of just `"text-icon"` here?

Reply:
Nope. Good catch :+1: